### PR TITLE
subversion: fix krb5 call

### PR DESCRIPTION
**Pull request by syarra**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

I made a change at line 90 to make it work, but I don't know how to generalize it using homebrew `ENV` (environment) variables.

### CODE REVIEW

File: Formula/subversion.rb [hunk 1]
```diff
@@ -87,7 +87,7 @@ def install
       ENV.universal_binary if build.universal?
 
       # scons ignores our compiler and flags unless explicitly passed
-      args = %W[PREFIX=#{serf_prefix} GSSAPI=/usr CC=#{ENV.cc}
+      args = %W[PREFIX=#{serf_prefix} GSSAPI=/home/syarra/.linuxbrew/ CC=#{ENV.cc}
```
Comment (sjackman) [inline Formula/subversion.rb]: Does `GSSAPI=#{Formula["krb5"].prefix}` work for you?

Comment (syarra) [inline Formula/subversion.rb]: Yes it does. I updated the PR. Thanks

File: Formula/subversion.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@ class Subversion < Formula
   depends_on :apr => :build
 
   resource "serf" do
-    url "https://serf.googlecode.com/svn/src_releases/serf-1.3.8.tar.bz2", :using => :curl
+    url "https://archive.apache.org/dist/serf/serf-1.3.8.tar.bz2", :using => :curl
```
Comment (sjackman) [inline Formula/subversion.rb]: Can you please remove `, :using => :curl`?
```
2.00s==> brew audit subversion
Error: 1 problem in 1 formula
subversion:
  * Stable resource "serf": Redundant :using value in URL
==> FAILED
```
https://travis-ci.org/Linuxbrew/homebrew-core/builds/143556238

**Comment by sjackman** [comment]
Thanks for your contribution, Sylvain!

**Comment by syarra** [comment]
You're welcome!
Thanks to you to for your help.

**Pull request merged**